css/main.css

```diff
@@ -0,0 +1,603 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.mobile-nav ul {
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  width: 58%;
+}
```

Comment:
Please try to follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838) as closely as possible 

You can try the following code to style the mobile-nav:
```
.mobile-nav ul {
  display: flex;
  flex-direction: column;
  gap: 1rem;
}

.mobile-nav li {
  text-align: left;
}

.mobile-nav a {
  padding-left: 20px;
}
```